core/src/main/java/org/elasticsearch/rest/action/cat/RestAllocationAction.java

```diff
@@ -68,7 +68,7 @@ public void doRequest(final RestRequest request, final RestChannel channel, fina
             @Override
             public void processResponse(final ClusterStateResponse state) {
                 NodesStatsRequest statsRequest = new NodesStatsRequest(nodes);
-                statsRequest.clear().fs(true);
+                statsRequest.clear().fs(true).indices(true);
```

Comment:
If the `store` is the only thing we need, we could set `Flag.store` flag for `indices` here.